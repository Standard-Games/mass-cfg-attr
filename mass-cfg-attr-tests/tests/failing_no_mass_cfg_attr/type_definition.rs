#![deny(unused_must_use, deprecated)]
#![allow(path_statements)] // disable warning for test

#[deprecated]
type TypeDefinition = ();

fn main() {
    // Violation of unused_must_use if not removed by mass_cfg_attr
    let _: TypeDefinition = ();
}
