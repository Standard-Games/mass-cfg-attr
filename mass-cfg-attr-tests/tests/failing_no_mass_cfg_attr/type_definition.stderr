error: use of deprecated type alias `TypeDefinition`
 --> tests/failing_no_mass_cfg_attr/type_definition.rs:9:12
  |
9 |     let _: TypeDefinition = ();
  |            ^^^^^^^^^^^^^^
  |
note: the lint level is defined here
 --> tests/failing_no_mass_cfg_attr/type_definition.rs:1:26
  |
1 | #![deny(unused_must_use, deprecated)]
  |                          ^^^^^^^^^^
